src-db/database/sourcedata/OBUISEL_SELECTOR_FIELD.xml

```diff
@@ -1,22 +1,23 @@
 <?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This file was uploaded without format from another issue. It appeared to me after I exported my database changes, so I uploaded it in this